src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -331,6 +334,77 @@ struct ScalarOp<ONNXTanOp> {
   using IOp = NotSuportedScalarOp;
 };
 
+//===----------------------------------------------------------------------===//
+// Scalar unary ops for lowering ONNXGeluOp
+//===----------------------------------------------------------------------===//
+template <>
+struct ScalarOp<ONNXGeluOp> {
+  using FOp = CustomScalarOp;
+  using IOp = CustomScalarOp;
+};
+
+template <>
+double analyzeSimdFor<ONNXGeluOp>(
+    Type t, Operation *op, int64_t &von, int64_t &son) {
+  double results;
+  StringRef approximate = dyn_cast<ONNXGeluOp>(op).getApproximate();
+  if (approximate.equals_insensitive("none"))
+    results = simdAnalysis(
+        {GenericOps::ArithmeticGop, GenericOps::MulGop}, {1, 3}, t, von, son);
```

Comment:
Please add `GenericOps:: ErfGop` with 1

Reply:
Thanks! I did not realize we had a `GenericOps:: ErfGop`. Let me fix that now

Reply:
Done!